conformance-tests/conformance_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
shall we add a build flag here like `//go:build conformance` so that the tests are not built during regular test run?  

this means ` -skip TestConformance` will also be not necessary

However, when running conformance tests, it will be necessary to add `-tags conformance`:

```
go test ... -tags conformance
```